interface/wx/spinctrl.h

```diff
@@ -214,6 +221,13 @@ class wxSpinCtrl : public wxControl
         Calling this method doesn't generate any @c wxEVT_SPINCTRL events.
     */
     void SetValue(int value);
+
+    /**
+        Sets the increment for the control, just like with wxSpinCtrlDouble
```

Comment:
Does it really make sense to speak of `wxSpinCtrlDouble` here? The increment could be explained just in terms of the change which happens when arrows are used.

Reply:
OK